src/Executor/Run.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Diagnostics;
+
+namespace Microsoft.DotNet.Execute
+{
+    public class Run
```

Comment:
It's sort of odd that this is an instance class. The class doesn't store any meanginful state, and has only one method.  Could it be static, so that you didn't have to say "new Run()", and just have Run.ExecuteProcess?